src/main/java/org/folio/marc/migrations/domain/entities/types/RecordState.java

```diff
@@ -0,0 +1,5 @@
+package org.folio.marc.migrations.domain.entities.types;
+
+public enum RecordState {
+  ACTUAL, DRAFT, OLD, DELETED
+}
```

Comment:
Is it an enum for persistence? If not let's move it from "entities" package.
Can't we have only "ACTUAL" records based on view query?